builtin/common/misc_helpers.lua

```diff
@@ -666,12 +669,12 @@ function core.translate(textdomain, str, ...)
 	local end_seq = ESCAPE_CHAR .. "E"
 	local arg_index = 1
 	local translated = str:gsub("@(.)", function(matched)
-		local c = string.byte(matched)
-		if string.byte("1") <= c and c <= string.byte("9") then
-			local a = c - string.byte("0")
+		local c = string_byte(matched)
+		if string_byte("1") <= c and c <= string_byte("9") then
+			local a = c - string_byte("0")
 			if a ~= arg_index then
-				error("Escape sequences in string given to core.translate " ..
-					"are not in the correct order: got @" .. matched ..
+				error("Escape sequences in string given to core.translate "  ..
+					"are not in the correct order: got @" .. matched  ..
```

Comment:
One space should be enough, why add another one?

Reply:
Auto formatting is not perfect. I will chefk it all manually and fix it.
Thanks for the help!